notebooks/article_1.py

```diff
@@ -0,0 +1,1201 @@
+#
+# It's essential to understand how to transform raw data into meaningful information. To achieve this, we will follow the workflow below:
+#
+# 1. Data exploration
+# 2. Pre-processing
+# 3. Data analysis
+# 4. Data-driven conclusions
+#
+#
+# Here are the main objectives of our analysis:
+#
+# - Examine the industries where startup acquisitions are most common, highlighting how crucial the type of industry is.
+# - Examine whether reaching certain milestones increases a startup's likelihood of acquisition.
+# - Analyze the average total funding of startups that get acquired versus those that close.
+# - Evaluate if startups with Series A, B, or C funding are more likely to succeed than those backed by angel investors or VC.
+#
+# Before we start, let's investigate the startup related concepts to understand the key challenges in this ecosystem.
+#
+# ## Understanding the Startup Ecosystem
+# In the startup field, there are important concepts to understand. First, startups raise money in stages called **funding rounds**, like Series A, B, or C. These rounds enables the startups to invest in the necessary resources for their growth. Second, startups set goals called **milestones**, like launching a product or getting a certain number of users. These reflect their performance to the investors.
+#
+# Finally, startups secure funding from to types of source:
+# - **Angel investors** are high net worth individuals who provide personal capital for investment. - **venture capital firms** are corporate entities specializing in investing in startups.
+#
+# Let’s get in to the preliminary steps.
+
+# ## Data Exploration
+
+# Data exploration serves as the first step in data analysis, involving the examination of datasets to identify patterns and detect anomalies.
+#
+# The following libraries provide the necessary tools and functions for data manipulation, analysis, and visualization.
+
+# In[2]:
+
+
+import os
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import plotly.graph_objects as go
+import plotly.express as px
+from scipy.stats import zscore
+
+
+# Now, we load the data into a [Pandas](https://pandas.pydata.org/) DataFrame.
+
+# In[3]:
+
+
+file_path = os.path.join('..', 'saved_model', 'startup.csv')
+data_df = pd.read_csv(file_path, encoding="ISO-8859-1")
+
+
+# In[4]:
+
+
+data_df.head(2)
+
+
+# Next, let's focus on understanding the significance of the Target Variable.
+
+# The `status` variable is the most important feature in this dataset. This binary variable indicates whether a startup has been acquired or closed.
+#
+# When analyzing the variable, it shows that 597 startups in our dataset were acquired, while 326 startups closed down.
```

Comment:
It would be nice to have a simple visualization here to match the phrase: a simple bar plot with the count of acquired vs closing.